### DISCUSSION TITLE
Notification/New message icon?

### DISCUSSION THREAD

**Discussion by trueTrueman** [Tampermonkey/tampermonkey#1335] [Q&A]
Hi, I'm probably missing something obvious, but what does the number in a red circle mean that oftenly appears above the Tampermonkey extension icon, similar to those unread messages counters on the icons of messaging apps?
Like this: ![Screenshot_693](https://user-images.githubusercontent.com/53678740/134753093-d32c5724-5345-4875-84e2-680715c56431.png)

**Comment by derjanb** [comment]
The number shows the number of executed script instances. Please note that higher instance numbers are usually caused by iframes.

Please also see #163 and/or #446 how to modify or even disabled it.
